### DISCUSSION TITLE
How can I run it on macOS

### DISCUSSION THREAD

**Discussion by proteida** [truckersmp-cli/truckersmp-cli#348] [Show and tell]
To do this you need to install the Windows version of the game
The easiest way to do it:

1. Install [Whisky](https://github.com/Whisky-App/Whisky) 
2. Create a bottle and install Steam and the game if not already installed
3. Open terminal and run command:
```shell
export WINE=~/Library/Application\ Support/com.isaacmarovitz.Whisky/Libraries/Wine/bin/wine64
truckersmp-cli start ets2mp \
    -g ~/path/to/Euro\ Truck\ Simulator\ 2 \
    -x ~/Library/Containers/com.isaacmarovitz.Whisky/Bottles/YOUR-BOTTLE
```

**Comment by Lucki** [comment]
Thanks for sharing!
